packages/lesswrong/components/localGroups/LocalGroupPage.tsx

```diff
@@ -119,13 +120,25 @@ const LocalGroupPage = ({ classes, documentId: groupId }: {
               }
             </SectionFooter>
           </div>
-          <ContentItemBody
+          {group.contents && <ContentItemBody
             dangerouslySetInnerHTML={htmlBody}
             className={classes.groupDescriptionBody}
             description={`group ${groupId}`}
-          />
+          />}
```

Comment:
`ContentItemBody` was just adding extra space to the page when there was no group description, so I'm hiding it in that case